go/pkg/pgn/config/config.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2021 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	"github.com/scionproto/scion/go/lib/addr"
+	"github.com/scionproto/scion/go/lib/config"
+	"github.com/scionproto/scion/go/lib/env"
+	"github.com/scionproto/scion/go/lib/log"
+	"github.com/scionproto/scion/go/lib/serrors"
+)
+
+const (
+	DefaultDb = "./pgn.db"
+)
+
+type Config struct {
+	General  env.General `toml:"general,omitempty"`
+	Features env.Features
+	Logging  log.Config       `toml:"log,omitempty"`
+	Metrics  env.Metrics      `toml:"metrics,omitempty"`
+	Sciond   env.SCIONDClient `toml:"sciond_connection,omitempty"`
```

Comment:
>`Sciond   env.SCIONDClient `toml:"sciond_connection,omitempty"``

Try to follow the recommendations for the naming conventions from https://scion.docs.anapaya.net/en/latest/contribute/go-style.html